cmd/infra/azure/create.go

```diff
@@ -897,7 +897,7 @@ func execAzCommand(cmdStr string) ([]byte, error) {
 	cmd := exec.Command("sh", "-c", cmdStr)
 	output, err := cmd.CombinedOutput()
 	if err != nil {
-		return nil, fmt.Errorf("failed to execute az command: %w", err)
+		return nil, fmt.Errorf("failed to execute az command, output: %s err: %w", string(output), err)
```

Comment:
seems like we already return this output to the caller in an error below on 904?

Reply:
I removed that line now